src/main/java/de/tum/in/www1/artemis/web/rest/UserResource.java

```diff
@@ -274,10 +284,40 @@ public ResponseEntity<UserDTO> getUser(@PathVariable String login) {
     @PreAuthorize("hasRole('ADMIN')")
     public ResponseEntity<Void> deleteUser(@PathVariable String login) {
         log.debug("REST request to delete User: {}", login);
+        if (isCurrentUser(login)) {
+            throw new BadRequestAlertException("You cannot delete yourself", "userManagement", "cannotDeleteYourself");
+        }
         userService.deleteUser(login);
         return ResponseEntity.ok().headers(HeaderUtil.createAlert(applicationName, "userManagement.deleted", login)).build();
     }
 
+    /**
+     * Delete users: deletes the provided users
+     *
+     * @param logins user logins to delete
+     * @return the ResponseEntity with status 200 (OK)
+     */
+    @DeleteMapping("users")
+    @PreAuthorize("hasRole('ADMIN')")
+    public ResponseEntity<List<String>> deleteUsers(@RequestParam(name = "login") List<String> logins) {
+        log.debug("REST request to delete {} users", logins.size());
+        List<String> deletedUsers = new java.util.ArrayList<>();
+        for (String login : logins) {
+            try {
+                if (!isCurrentUser(login)) {
+                    userService.deleteUser(login);
+                    deletedUsers.add(login);
+                }
+            }
+            catch (Exception exception) {
+                // In order to handle all users even if some users produce exceptions, we catch them and ignore them and proceed with the remaining users
+                log.error("REST request to delete user {} failed", login);
+                log.error(exception.getMessage());
```

Comment:
```suggestion
                log.error(exception.getMessage(), exception);
```

Afaik the default impl would be this :) But that's what I know from other projects (I didn't have a look for this in Artemis :D )

Reply:
Yeah you are right :D thank you! Will check this